content/administration/maintain/neutralized_databases.rst

```diff
@@ -0,0 +1,36 @@
+=====================
+Neutralized databases
+=====================
+
+What is a neutralized database?
+===============================
+
+A neutralized database is a non-production database on which several parameters are deactivated. This enables one to
+carry out tests without the risk of launching specific automated processes that could impact production data (e.g.,
+sending emails to customers). Live access is then removed and turned into a testing environment.
+
+.. note::
+   **Any testing database created is a neutralized database:**
+   - testing backup databases
+   - duplicate databases
+   - For Odoo.sh: staging and development databases
```

Comment:
Lists unfortunately don't work inside alert blocks such as notes:
![image](https://github.com/odoo/documentation/assets/70884563/b4b024e9-71f0-4737-a978-237cafaabe1f)

Reply:
OK I replaced the hyphen with a comma instead

Reply:
Actually, lists do work (I tried), but an empty line is always required before starting a new list, my bad :)